### ISSUE TITLE
PDAL info function throws error "File not found" on remote files.

### ISSUE THREAD

**Issue by kylemann16** [PDAL/PDAL#2188]
When the info call reaches the makePipeline method in InfoKernel.cpp, if the file is not a local file fileExists returns false.
```
void InfoKernel::makePipeline(const std::string& filename, bool noPoints)
{
    if (!pdal::Utils::fileExists(filename))
        throw pdal_error("File not found: " + filename);
    ...
}
```
Example: ./bin/pdal info ept://http://na.entwine.io/red-rocks/ --debug
```
(PDAL Debug) Debugging...

PDAL: File not found: ept://http://na.entwine.io/red-rocks/
```

**Comment by abellgithub** [comment]
This is addressed along with #2170

**Comment by abellgithub** [comment]
This is addressed in #1894
